docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2023-11-15
```

Comment:
This is outdated, in any case it will have to wait for the decision to drop Nashorn use.